src/security_helper.cc

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "security_helper.h"
+
+#include "base/base64.h"
+
+#include "tweetnacl.h"
+#include <openssl/base64.h>
+#include <openssl/digest.h>
+#include <openssl/hkdf.h>
+#include <openssl/sha.h>
+
+namespace helper {
+
+std::string Security::Sign(
+    const std::vector<std::string> keys,
```

Comment:
I think you want these vectors (keys and values) to be const references.

Reply:
Not possible to my knowledge we can have vector<const T> until C++17?

Reply:
I'm just suggesting `const std::vector<std::string>& keys` rather than `const std::vector<std::string> keys`.

Reply:
👍 yeap missed this one, thanks